### ISSUE TITLE
Add rebased spec file to Bugzilla

### ISSUE THREAD

**Issue by torsava** [phracek/rebase-helper#199]
Hi,
I'd like to propose that the rebased spec file is then added to the Bugzilla report opened by upstream monitoring. Ideally accompanied by short bullet points listing the non-obvious modifications (if any).

It wouldn't matter it it's appended as a file to Bugzilla, or provided as a link somewhere.

This will be especially useful when issue #186 gets implemented, so that unaware maintainers are made aware of the change instead of having to do research.

Have a great day!

**Comment by thozza** [comment]
**Note:**
This has to be implemented in ```the-new-hotnes``` not in rebase-helper.

**Comment by phracek** [comment]
@thozza yes it has to be definitely implemented in the-new-hotness. But I thought, we can provide a reference to the rebased spec file via API.

**Comment by thozza** [comment]
@phracek the path to rebased SPEC could be in the JSON output or simply in the dictionary with results.
